lib/LedgerSMB/Routes/ERP/API/Languages.pm

```diff
@@ -274,7 +274,7 @@ paths:
       operationId: getLanguages
       responses:
         200:
-          description: ...
+          description: Returns the full set of configured languages
```

Comment:
openapi & info sections left purposely?

Reply:
No; accidentally left behind. Fix pushed. Thanks for the review!